src/commands/branches.ts

```diff
@@ -315,10 +343,16 @@ const create = async (
         if (!branch) {
           throw new Error(`Branch ${props.parent} not found`);
         }
-        return { parent_id: branch.id };
+        return { parent_id: branch.id, protected: branch.protected };
       });
   })();
 
+  if (parentProps.protected) {
+    process.stderr.write(
+      'The parent branch is protected, the new branch role password will be changed.',
+    );
+  }
```

Comment:
This should be a warning or info, and not an error. We have `src/log.ts` helper maybe use `log.info` or create `log.warning` 

Can we copy the warning message from creating a branch on the console? Maybe @bgrenon can help with the copy.

Reply:
added `log.warning`

Reply:
I'm also reaching out to Barry personally to approve the text.

Reply:
The text is updated.